### ISSUE TITLE
Reduce model size

### ISSUE THREAD

**Issue by prem6667** [facebookresearch/StarSpace#268]
Hi, 
    Is there any way to **reduce the size** of the trained modal? 
    In my training dataset have 1.4 million examples having around 2800 categories with one-to-many relationship. parameters used in training are as follow: 

- dim : 200
- batchSize : 16
- ngrams : 4
- bucket : 500000
- normalizeText : 1
- maxNegSamples : 5 
- ws : 8 

Resulting modal size is around **12GB**

**Comment by sudoaza** [comment]
You are using ngrams 4 without default mincount (1), so any 4 token combination is being saved. Use -minCount 5 or something
